### ISSUE TITLE
Realtime federation does not update containers list

### ISSUE THREAD

**Issue by Amantini1997** [3drepo/3drepo.io#4035]
### Description

Highlighting objects on a federation node that would highlight multiple sub models is not functioning.

- This is happening in: both
- This requires a hotfix to production: no

### Steps to replicate
- Open the same teamspace with 2 different accounts
- edit the containers list for a federation

### Current Behaviour
 the containers on the same federation in the other account stay the same

### Expected Behaviour
 the containers on the same federation in the other account should update

### Notes
Use `useEffect` on the `federation.revision._id` to detect an update in the containers list.
You may need to wait for this to be merged in https://github.com/3drepo/3drepo.io/pull/4028 (or to apply the same changes) in order to be able to use a federation revision

**Comment by keifgwinn** [comment]
I've also observed similar behaviour on containers when using the bulk uploading scripts. If I'm logged into the account, and use the API key to create containers and upload models, they don't appear in my UI until a forced refresh.

**Comment by carmenfan** [comment]
> I've also observed similar behaviour on containers when using the bulk uploading scripts. If I'm logged into the account, and use the API key to create containers and upload models, they don't appear in my UI until a forced refresh.

Wouldn't say that is related to this.

It's a different event all together to see new containers. If you can spare a minute to create a separate issue that would be great

**Comment by Amantini1997** [comment]
> > I've also observed similar behaviour on containers when using the bulk uploading scripts. If I'm logged into the account, and use the API key to create containers and upload models, they don't appear in my UI until a forced refresh.
> 
> Wouldn't say that is related to this. (Unless you're talking about the federation not showing the list of containers updated)
> 
> It's a different event all together to see new containers. If you can spare a minute to create a separate issue that would be great

You are right. We were talking about this earlier and eventually I opened a new issue for that https://github.com/3drepo/3drepo.io/issues/4038

**Comment by carmenfan** [comment]
@Amantini1997  is this fixed by #4028?

**Comment by Amantini1997** [comment]
> @Amantini1997 is this fixed by #4028?

Nope. Upon updating the list of containers for a federation, you can see the same fed in the other browser not being updated
![image](https://user-images.githubusercontent.com/43547703/223406776-3e1211c5-8ae5-47c1-b922-be081ae820aa.png)

**Comment by Amantini1997** [comment]
This is fixed, not sure at which point though
